cloudfoundry-client-reactor/src/test/java/org/cloudfoundry/reactor/uaa/authorizations/ReactorAuthorizationsTest.java

```diff
@@ -97,7 +97,7 @@ public void authorizeByAuthorizationCodeGrantBrowser() {
     public void authorizeByAuthorizationCodeGrantHybrid() {
         mockRequest(InteractionContext.builder()
             .request(TestRequest.builder()
-                .method(GET).path("/oauth/authorize?client_id=app&redirect_uri=http://localhost:8080/app/&scope=openid&response_type=code%20id_token")
+                .method(GET).path("/oauth/authorize?client_id=app&redirect_uri=http%3A%2F%2Flocalhost%3A8080%2Fapp%2F&scope=openid&response_type=code%20id_token")
```

Comment:
I don't particularly like how this turned out, but tests like this one already had some encoded characters even before my change. See `org.cloudfoundry.reactor.uaa.clients.ReactorClientsTest.list()`.